xbmc/utils/StreamDetails.cpp

```diff
@@ -584,12 +584,15 @@ std::string CStreamDetails::VideoDimsToResolutionDescription(int iWidth, int iHe
 
   else if (iWidth <= 720 && iHeight <= 480)
     return "480";
-  // 720x576 (PAL) (768 when rescaled for square pixels)
-  else if (iWidth <= 768 && iHeight <= 576)
-    return "576";
   // 960x540 (sometimes 544 which is multiple of 16)
   else if (iWidth <= 960 && iHeight <= 544)
     return "540";
+  // NTSC DVD
+  else if (iWidth <= 1024 && iHeight <= 480)
```

Comment:
854x480 for NTSC anamorphic

Reply:
So you mean modifying this https://github.com/xbmc/xbmc/blob/master/xbmc/utils/StreamDetails.cpp#L585

to: `else if (iWidth <= 854 && iHeight <= 480)`

?

Reply:
yes exactly.

Reply:
done

Reply:
almost but not there yet, sent details via Slack